endtoendtests/Azure.Functions.Java.Tests.E2E/Azure.Functions.Java.Tests.E2E/HttpEndToEndTests.cs

```diff
@@ -33,5 +34,20 @@ public async Task HttpTrigger_ReturnsCustomCode()
         {
             Assert.True(await Utilities.InvokeHttpTrigger("HttpTriggerCustomCode", "?&name=Test",  HttpStatusCode.OK, "Test", 209));
         }
+
+        [Fact]
+        public async Task HttpTriggerJavaClassLoader()
```

Comment:
Which scenario would you want to validate? 
According to the code, this test validates, if the environment variables "AzureWebJobsStrage" is exists or not when the java_home does not contain "zulu-11" and "JAVA_REVERSE_LIB_LOADING" is false or null.  In this case, you are changing the order of the class loading. So the test should be valid if you change the "JAVA_REVERSE_LIB_LOADING"  configuration, the order is changed properly.

Reply:
I will more explanation sir